helm_deploy/values-prod.yaml

```diff
@@ -15,10 +15,11 @@ generic-service:
     PRISON_SERVICE_BASE_URL: https://api.prison.service.justice.gov.uk
 
 generic-prometheus-alerts:
-  sqsAlertsQueueNames:
+  sqsOldestAlertQueueNames:
     - "hmpps-person-record-prod-cpr_court_case_events_queue"
+    - "hmpps-person-record-prod-cpr_delius_offender_events_queue"
+  sqsNumberAlertQueueNames:
     - "hmpps-person-record-prod-cpr_court_case_events_dlq"
     - "hmpps-person-record-prod-cpr_delius_offender_events_dlq"
-    - "hmpps-person-record-prod-cpr_delius_offender_events_queue"
-    # - "Digital-Prison-Services-prod-cpr_offender_events_dlq" omitting for now
-    # - "Digital-Prison-Services-prod-cpr_offender_events_queue" # we are not yet consuming from this queue
+  sqsAlertsOldestThreshold: 15
+  sqsAlertsTotalMessagesThreshold: 1
```

Comment:
Tell us when any messages we can't process are received. 
Also tells us if messages are building up on one of the queues we are consuming from